configs/sites/noaa-aws/packages.yaml

```diff
@@ -41,6 +41,13 @@ packages:
     externals:
     - spec: diffutils@3.3
       prefix: /usr
+  ecflow:
+    buildable: False
+    externals:
+    - spec: ecflow@5.8.4+ui+static_boost
+      prefix: /data/prod/jedi/spack-stack/ecflow-5.8.4
```

Comment:
is this the correct prefix for noaa-aws?

Reply:
No, it is not. Thanks for spotting that. Let me update from the site config.